src/components/track-changes/suggestion-list/SuggestionSnippet.tsx

```diff
@@ -42,115 +48,152 @@ export const SuggestionSnippet: React.FC<{ suggestion: TrackedChange }> = ({
     doc: store.doc,
   }))
   const [snippet, setSnippet] = useState<SnippetData | null>(null)
-  const [message, setMessage] = useState('')
   const { dataTracked } = suggestion
+  const contentRef = useRef<HTMLDivElement>(null)
+  const isContainLaTeX = (text: string): boolean => {
```

Comment:
this probably doesn't need to be defined in the context of react component and can be move to lib

Reply:
I have changed the implementation